pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -244,31 +208,34 @@ pub mod pallet {
 	pub type RelayerList<T: Config> =
 		StorageDoubleMap<_, Blake2_128Concat, Domain, Blake2_128Concat, DomainAddress, ()>;
 
-	#[pallet::storage]
-	#[pallet::getter(fn outbound_message_nonce_store)]
-	pub type OutboundMessageNonceStore<T: Config> =
-		StorageValue<_, T::OutboundMessageNonce, ValueQuery>;
-
-	/// Storage for outbound messages that will be processed during the
-	/// `on_idle` hook.
-	#[pallet::storage]
-	#[pallet::getter(fn outbound_message_queue)]
-	pub type OutboundMessageQueue<T: Config> = StorageMap<
-		_,
-		Blake2_128Concat,
-		T::OutboundMessageNonce,
-		(Domain, T::AccountId, T::Message),
-	>;
-
-	/// Storage for failed outbound messages that can be manually re-triggered.
-	#[pallet::storage]
-	#[pallet::getter(fn failed_outbound_messages)]
-	pub type FailedOutboundMessages<T: Config> = StorageMap<
-		_,
-		Blake2_128Concat,
-		T::OutboundMessageNonce,
-		(Domain, T::AccountId, T::Message, DispatchError),
-	>;
+	//TODO(cdamian): Migration?
+	// #[pallet::storage]
+	// #[pallet::getter(fn outbound_message_nonce_store)]
+	// pub type OutboundMessageNonceStore<T: Config> =
+	// 	StorageValue<_, T::OutboundMessageNonce, ValueQuery>;
+
+	//TODO(cdamian): Migration?
+	// /// Storage for outbound messages that will be processed during the
+	// /// `on_idle` hook.
+	// #[pallet::storage]
+	// #[pallet::getter(fn outbound_message_queue)]
+	// pub type OutboundMessageQueue<T: Config> = StorageMap<
+	// 	_,
+	// 	Blake2_128Concat,
+	// 	T::OutboundMessageNonce,
+	// 	(Domain, T::AccountId, T::Message),
+	// >;
+
+	//TODO(cdamian): Migration?
+	// /// Storage for failed outbound messages that can be manually re-triggered.
+	// #[pallet::storage]
+	// #[pallet::getter(fn failed_outbound_messages)]
+	// pub type FailedOutboundMessages<T: Config> = StorageMap<
+	// 	_,
+	// 	Blake2_128Concat,
+	// 	T::OutboundMessageNonce,
+	// 	(Domain, T::AccountId, T::Message, DispatchError),
+	// >;
```

Comment:
Hmm, depends on how the RU update happens. I would imagine that it would happen after a block is produced, thus after an `on_idle` hook is called.

Reply:
I meant over a human period of time. Basically we need to know if we need to maintain the message state because the users are using it at that moment or we can just assume that they will not be using it when the RU is enacted.

Reply:
Asumming there're no storage or just purge it

Reply:
@wischli, how do you see adding this to the current RU? THat way we avoid future migration problems because now we need to ensure we have not message there.

If we go with it, just we need to upgrade the version number of this pallet. And adding a reset migration to clean the Nonce storage. WDYT? Of course, I do not want to delay the RU, just if we have time to do it.

Reply:
Forgot the above, I think we can ensure the same conditions easily in the next RU

Reply:
My understanding was that we can assume there won't be any pending LP messages when we trigger the execution of the update. However, after triggering the RU, it takes one hour for it to be live. Right now, we cannot block those permissionless extrinsics which create outgoing messages. So theoretically, there is a chance. Let's discuss this in the Eng sync today.